common/src/main/java/com/sequenceiq/cloudbreak/quartz/JobDataMapProvider.java

```diff
@@ -0,0 +1,35 @@
+package com.sequenceiq.cloudbreak.quartz;
+
+import org.quartz.JobDataMap;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.quartz.model.JobResourceAdapter;
+
+@Component
+public class JobDataMapProvider {
```

Comment:
this component is not injected anywhere

Reply:
It is injected here: com/sequenceiq/cloudbreak/job/stackpatcher/ExistingStackPatcherJobService.java:43

Reply:
my bad, did not recognize